app/uk/gov/hmrc/agentsubscriptionfrontend/views/postcode.scala.html

```diff
@@ -50,7 +50,9 @@
             labelAsHeading = true,
             labelClass = Some("govuk-label--xl"),
             hint = Some(msgs(s"postcode.${businessType.key}.hint")),
-            inputClass = Some("govuk-input--width-10")
+            hintId = Some("postcode-hint"),
+            inputClass = Some("govuk-input--width-10"),
+            id = "postcode"
```

Comment:
You shouldn't need to feed id & hintId here because govukInput uses the `.withFormField(field)` which creates the id and hintId... unless the tests say otherwise?

Reply:
yeah. so in the test i am testing that the hint and the label have the correct text. which is why I added them.

Reply:
that's weird though because the tests are taking the hmtl output, so I would still expect the hint and id to be rendered by the component with just the `.withFormField(field)` method - so you're saying that without this they aren't behaving like we expect from   https://github.com/hmrc/play-frontend-hmrc#withformfield  ?